project-maintainers.csv

```diff
@@ -738,6 +738,47 @@ Incubating,Emissary-Ingress,Rafael Schloming,Ambassador Labs,rhs,https://github.
 ,,Flynn,Ambassador Labs,kflynn,
 ,,John Esmet,Ambassador Labs,esmet,
 ,,Alix Cook,Ambassador Labs,acookin,
+Incubating,Cilium,Aditi Ghag,Isovalent,aditighag,https://github.com/cilium/cilium/blob/master/MAINTAINERS.md
+,,Alexandre Perrin,Isovalent,kaworu
```

Comment:
Nit: I think a comma is missing at the end of each line except for the first since the last column should be the URL to the maintainers file. GitHub can't render the file as a result:
![image](https://user-images.githubusercontent.com/1764210/142765079-d1cb468a-4473-4cb0-a27c-786bf26440c3.png)